wp_api/src/templates.rs

```diff
@@ -160,7 +161,7 @@ pub struct SparseTemplate {
     #[WpContext(edit, embed, view)]
     pub author: Option<UserId>,
     #[WpContext(edit, view)]
-    pub modified: Option<bool>,
+    pub modified: Option<BoolOrWpGmtDateTime>,
```

Comment:
Based on [the API doc](https://developer.wordpress.org/rest-api/reference/wp_templates/), it looks like we should treat boolean values (which I assume are always `false`) as an `Option<WpGmtDateTime>.none`.

> The date the template was last modified, in the site's timezone.
> JSON data type: string,
> Format: datetime ([details](https://core.trac.wordpress.org/ticket/41032))